_maps/configs/independent_shetland.json

```diff
@@ -22,8 +22,8 @@
 			"officer": true,
 			"slots": 1
 		},
-		"Lieutenant": {
-			"outfit": "/datum/outfit/job/lieutenant",
+		"First Officer": {
```

Comment:
wait why does the shetland have both a lieutenant/first officer and a first mate anyways
just yeet one of them

Reply:
Oh I didn't see the first mate slot when I was doing this. Whoops.